payload/sp/ShaUtil.cc

```diff
@@ -0,0 +1,18 @@
+#include "ShaUtil.hh"
+#include <vendor/libhydrogen/hydrogen.h>
```

Comment:
While you're fixing includes:
```suggestion
#include "ShaUtil.hh"

#include <vendor/libhydrogen/hydrogen.h>
```